firstvoices/backend/tests/test_apis/base/base_async_api_test.py

```diff
@@ -137,7 +137,9 @@ class AsyncWorkflowTestMixin:
     """Use with BaseSiteContentApiTest, WriteApiTestMixin,"""
 
     @pytest.mark.django_db
-    @pytest.mark.parametrize("job_status", [JobStatus.COMPLETE, JobStatus.STARTED])
+    @pytest.mark.parametrize(
+        "job_status", [JobStatus.ACCEPTED, JobStatus.COMPLETE, JobStatus.STARTED]
```

Comment:
Looks like this adds a new restriction on the dictionary-cleanup and bulk-visibility APIs-- was that intentional?

Reply:
Oh whoops you're right good catch, I'll revert this change so it only affects import jobs